internal/output/plain.go

```diff
@@ -22,6 +22,27 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+// PrependedSliceItem wraps an output in a slice and prepends it with a custom string
+type PrependedSliceItem struct {
+	v       interface{}
+	prepend string
+}
+
+// NewPrependedSliceItem wraps an output item in a slice and prepend its output with a custom string instead of a bullet point
+func NewPrependedSliceItem(v interface{}, prepend string) PrependedSliceItem {
+	return PrependedSliceItem{v, prepend}
+}
+
+// PrependString returns the prepended string
+func (po PrependedSliceItem) PrependString() string {
+	return po.prepend
+}
+
+// Value returns the wrapped value
+func (po PrependedSliceItem) Value() interface{} {
+	return po.v
+}
```

Comment:
This is again adding new convention and it seems like all this is doing is prepending a value to a string? I don't want to see any updates to the output package for this PR.